### DISCUSSION TITLE
Bridge config not parsed in config file

### DISCUSSION THREAD

**Discussion by tam481** [t0bst4r/home-assistant-matter-hub#127] [General]
Hi @t0bst4r 
Thank you very much for including support for a config file. I am trying to configure my bridge and it seems to be OK parsing the initial parameters until it gets to name, port and filter and it stops and says "Unknown arguments: name, port, filter"

I thought I would raise this here first in case it is not actually an issue although it is for me :-)

{
    "homeAssistantUrl": "REDACTED",
    "homeAssistantAccessToken": "REDACTED",
    
    "logLevel": "debug",

    "name": "HASS-Matterhub",
    "port": 5540,
    "filter": {
        "include": [
            {
                "type": "label",
                "value": "exposed_to_voice_assistant"
            },
            {
                "type": "domain",
                "value": "light"
            }
        ],
        "exclude": [
            {
                "type": "label",
                "value": "unavailable_ok"
            }
        ]
}

**Comment by darth-hp** [comment]
The closing } for filter is missing

**Comment by tam481** [comment]
@darth-hp You're right. I truncated my actual config file because it's irrelevant to be honest. The error appears to indicate that the application is not recognising the parameters hence the "Unknown arguments: name, port, filter" message.

**Comment by t0bst4r** [comment]
Sorry, I didn’t see the discussion earlier.
The config file only includes the general settings which are relevant for the whole app (See configuration section in the readme). 

Bridge configuration needs to be done in the UI and is independent from ha token or logLevel etc. 

Of course in the end bridge settings are also stored as JSON files in the data directory, but they are not meant to be edited by hand. But would be possible of course.
